sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ### 4.3.0b3 (Unreleased)
 
+>[!WARNING]
```

Comment:
It doesn't look like this heading is rendering quite right?

Reply:
It looks like this on my rendering, which I think is what we're looking for:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/30335873/156472477-8e7b036a-87bd-487c-926d-7fb84a47b9db.png">